resume.html

```diff
@@ -0,0 +1,95 @@
+<!DOCTYPE html>
+
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Portfolio: Ashley Whitted</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://fonts.googleapis.com/css?family=Dancing+Script|Lobster|Sacramento|Satisfy|Tangerine" rel="stylesheet">
```

Comment:
Loading all these fonts will definitely have an impact on the load time of your page -- you'd want to remove any fonts you don't absolutely need for the display of the page.